index.html

```diff
@@ -2205,7 +2205,7 @@ <h2>Definition of Roles</h2>
 					</tr>
 					<tr>
 						<th class="role-properties-head" scope="row">Supported States and Properties:</th>
-						<td class="role-properties">					
```

Comment:
ditto: remove irrelevant whitespace changes. If you want to do a whitespace change, make it part of a standalone PR, so it doesn't break blame.